docs/manual/developer/06_contributing_with_content.md

```diff
@@ -1268,6 +1268,21 @@ the following to `rule.yml`:
 
 -   Languages: OVAL
 
+#### coreos_kernel_option
+-   Checks that `argument=value` pair is present in the kernel arguments.
+    Note that this applies to Red Hat CoreOS.
+
+-   Parameters:
+
+    -   **arg_name** - Argument name, eg. `audit`
+
+    -   **arg_value** - Argument value, eg. `'1'`
+
+    -   **negate** - negates the check, which then ensures that
```

Comment:
```suggestion
    -   **arg_negate** - negates the check, which then ensures that
```

The actual parameter name is `arg_negate` although I don't know which of them makes more sense, probably something more verbose like `negate_check` or something

Reply:
But I'm fine with `arg_negate` as well